packages/nextjs/components/scaffold-eth/Contract/utilsContract.tsx

```diff
@@ -47,31 +47,33 @@ const getAllContractFunctions = (contract: Contract): FunctionFragment[] => {
  * which has corresponding input field for param type and button to read
  */
 const getContractVariablesAndNoParamsReadMethods = (
-  contract: Contract,
+  contract: Contract | null,
   contractMethodsAndVariables: FunctionFragment[],
   refreshDisplayVariables: boolean,
 ): { loaded: boolean; methods: (JSX.Element | null)[] } => {
-  return {
-    loaded: true,
-    methods: contractMethodsAndVariables
-      .map((fn, index) => {
-        const isQueryableWithNoParams =
-          (fn.stateMutability === "view" || fn.stateMutability === "pure") && fn.inputs.length === 0;
-        if (isQueryableWithNoParams) {
-          return (
-            // DV -> DisplayVariables
-            <DisplayVariable
-              key={`DV_${fn.name}_${index}`}
-              functionFragment={fn}
-              contractAddress={contract.address}
-              refreshDisplayVariables={refreshDisplayVariables}
-            />
-          );
-        }
-        return null;
-      })
-      .filter(n => n),
-  };
+  return contract
+    ? {
+        loaded: true,
```

Comment:
why do we need that flag?

Reply:
check https://github.com/scaffold-eth/se-2/pull/50

Reply:
strange variable, it's always true. Will check it later

Reply:
https://github.com/scaffold-eth/se-2/pull/100/commits/964bc20b8ca09fcae0b88743942acbf89442c8b4